runtime/drive_handler.go

```diff
@@ -125,10 +125,20 @@ func (h *stubDriveHandler) createStubDrive(driveID, path string) error {
 	return nil
 }
 
+func filterOnlyStubDrives(drives []models.Drive) []models.Drive {
+	stubDrives := make([]models.Drive, 0)
```

Comment:
nit: I'd rather prefer preallocating here as we know the size in advance: `make([]models.Drive, 0, len(drives))`

Reply:
Wouldn't it allocate more than what we need?

Reply:
Should it be just `make([]models.Drive)` then?

Reply:
We don't have the filter logic anymore, and all `make` calls explicitly specify the capacity.